7.0.0/oracle/identity_db_cleanup.sh

```
@@ -0,0 +1,132 @@
+      echo "$TENANT_ID" >> "$SUCCESSFUL_DELETIONS_FILE"
+    else
+      FAILED_LOG="$LOG_DIR/failed_tenant_${TENANT_ID}.log"
+      { echo "--- SQL ---"; echo "$DELETE_PROCEDURE"; echo "--- Output ---"; echo "$SQLPLUS_OUTPUT"; } > "$FAILED_LOG"
+      echo "Failed to process tenant ID $TENANT_ID with org ID $ORG_UUID. Check log: $FAILED_LOG" | tee -a "$SUMMARY_FILE"
+    fi
+  else
+    echo "Skipping invalid or empty line: TENANT_ID=$TENANT_ID, ORG_UUID=$ORG_UUID"
+  fi
+done < "$EXPORT_FILE"
+
+echo "Data deletion process completed. Summary available at $SUMMARY_FILE."
```

Shellcheck (0.11.0)

[error] Literal carriage return. Run script through tr -d '\r' .

(SC1017)